x-pack/plugins/siem/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "license": "Elastic-License",
   "scripts": {
-    "extract-mitre-attacks": "node scripts/extract_tactics_techniques_mitre.js & node ../../../scripts/eslint ../../legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts --fix",
+    "extract-mitre-attacks": "node scripts/extract_tactics_techniques_mitre.js && node ../../../scripts/eslint ../../legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts --fix",
```

Comment:
@XavierM @FrankHassanabad With this fix, when I run the script I get a single change:

```diff
diff --git a/x-pack/legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts b/x-pack/legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts
index 16ab733652..160e006c4d 100644
--- a/x-pack/legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts
+++ b/x-pack/legacy/plugins/siem/public/pages/detection_engine/mitre/mitre_tactics_techniques.ts
@@ -187,7 +187,7 @@ export const tacticsOptions: MitreTacticsOptions[] = [
   },
 ];
 
-export const technique = [
+export const techniques = [
   {
     name: '.bash_profile and .bashrc',
     id: 'T1156',

```

which looks to be because #55883 changed the generated file but not the script itself. Should I just update the script to match?

Reply:
That is true, you want the name `technique` without an `s`. I was rushed to get that fix in before FF and forgot to update and run these scripts.